tests/test_request_timeout.py

```diff
@@ -58,18 +58,36 @@ def close(self):
             t = req.loop.time()
             print("sending at {}".format(t), flush=True)
             conn = next(iter(args)) # first arg is connection
-            try:
-                delayed_resp = self.orig_send(*args, **kwargs)
-            except Exception as e:
-                return aiohttp.ClientResponse(req.method, req.url)
+            if aiohttp.__version__ >= "3.1.0":
```

Comment:
Why would this be greater than 3.1.0 given https://github.com/channelcat/sanic/pull/1180/files#diff-f4c265073ea4274a80eca85440875e56R2

🤔 

Whatever the case since your tests are passing, this is a good sign. They've all been failing lately 😢

Reply:
good question.
It is because tox.ini chooses `aiohttp>=2.3`
https://github.com/channelcat/sanic/blob/8a07463a67c61dd6303f3bf222600f0c5656565d/tox.ini#L15
So that is why tests were passing on my local machine with `aiohttp==1.3.5` as per requirements-dev.txt, but failing on Travis in tox virtual environment.
I changed the requirements-dev.txt to match what tox uses, and then I started seeing these problems with latest aiohttp at 3.1.0.

The other option would be simply pin requirements-dev.txt and tox.ini to `aiohttp==2.3.0`, so we don't have to deal with changes in new versions of aiohttp.

Reply:
That makes sense, thanks for the explanation. I think it's ok to leave the `>=`, at least for now. I like living on the cutting edge 😄